brjs-sdk/sdk/libs/javascript/br-ie8-shim/ie8-shim.js

```diff
@@ -47,3 +47,9 @@ if (Function.prototype.bind && typeof console == "object" && typeof console.log
 		console[method] = Function.prototype.call.bind(console[method], console);
 	}
 }
+
+if(!Number.isNaN) {
+	Number.isNaN = Number.isNaN || function(value) {
+	    return typeof value === "number" && value !== value;
```

Comment:
@dchambers Out of interest why didn't you use `isNan()` (https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/isNaN) here?